spec/factories/comments.rb

```diff
@@ -3,6 +3,7 @@
 FactoryBot.define do
   factory :comment do
     user
+    likes_count { Faker::Number.between(from: 1, to: 100) }
```

Comment:
This number should be related to the amount of likes, I'm not sure it should be set this way on the factory.